contracts/connectors/compound/main.sol

```diff
@@ -47,12 +49,13 @@ abstract contract CompoundResolver is Events, Helpers {
     */
     function withdraw(
         address token,
+        string calldata tokenId,
```

Comment:
@mubaris We are going to pass only tokenId. And from tokenId we can get ctoken and token address